build.gradle

```diff
@@ -36,12 +42,46 @@ allprojects {
     repositories {
         google()
         mavenCentral()
-        jcenter()
         mavenLocal()
+        jcenter() {
+            content {
+
+                // Remove when https://github.com/google/ExoPlayer/issues/5246 is closed
+                includeGroup("com.google.android.exoplayer")
+
+                // Remove when https://github.com/facebook/flipper/issues/1775 is closed
+                includeGroup("com.facebook.flipper")
+
+                // Remove when https://github.com/Kotlin/dokka/issues/41 is closed
+                includeGroup("org.jetbrains.dokka")
```

Comment:
Isn't it enough to list Dokka above, in the buildscript repositories section?

Reply:
It is needed because the plugin is added to all modules, not only to the main one